schema.sql

```diff
@@ -330,6 +330,15 @@ CREATE TABLE IF NOT EXISTS `tile_store` (
   FOREIGN KEY (`house_id`) REFERENCES `houses` (`id`) ON DELETE CASCADE
 ) ENGINE=InnoDB;
 
+CREATE TABLE IF NOT EXISTS `live_casts` (
```

Comment:
Actually hashing that password may not be a good idea in some cases. If you hash it, you can't display the plaintext on a website, which is the main reason why you'd store that password in a database table in the first place (otherwise you can keep it in memory - player cannot cast beyond server's runtime).

Those passwords are temporary (as opposed to account passwords). There is no username to them. By their nature, they are EXPECTED to be shared with other people, which means people would not (re)use their sensitive passwords (which is why we would want to hash them, keeping them somewhat inaccessible even to the server owners). I'm sure people would soon come up with when to share the password on a website (within a guild maybe?). Hashing it makes it forever impossible.